docs/stencil-docs/localization/localization-tutorial.md

```diff
@@ -0,0 +1,121 @@
+# Localization Tutorial
+<div class="otp" id="no-index">
+
+### On this page
+- [Adding a language file](#adding-a-language-file)
+- [Creating translation keys](#creating-translation-keys)
+- [Updating browser settings](#updating-browser-settings)
+- [Related resources](#related-resources)
+
+</div>
+
+You can localize your Stencil theme for your desired target language. This tutorial describes how to localize a storefront in Spanish. By following this method, you can display a specific language based on the language selected in the viewer's browser. By the end of this tutorial, you will have the tools to localize most areas of your theme.
+
+### Prerequisites
+
+In this tutorial, we will use Stencil CLI and [Browsersync](https://github.com/bigcommerce/browser-sync) to serve up a live preview of a theme in development.
+
+To complete this tutorial, you should be familiar with the following concepts:
+
+* [Installing Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil)
+* [Creating a Stencil CLI Token](https://support.bigcommerce.com/s/article/Store-API-Accounts)
+* [Downloading and uploading custom themes](https://support.bigcommerce.com/s/article/Stencil-Themes#download-upload)
+* [Serving a live preview](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/live-previewing-a-theme#serving-a-live-preview)
+
+
+## Adding a language file
+1. After downloading and extracting your Stencil theme, open the folder containing your theme files and navigate to the `/lang` subfolder. 
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme:  -->
+### Note
+
+The `/lang` subfolder includes the `en.json` file and example language files. The `en.json` file and your language file must be present for a localized Stencil theme to work. Each language requires its own JSON file. 
+
+</div>
+</div>
+</div>
+
+2. Name your translation file based on the [BCP 47 specification](https://tools.ietf.org/html/bcp47) of language and region codes.
```

Comment:
Need to mention the `es.json` file in this step being created and added to the `/lang` subfolder.